src/Shopsys/ShopBundle/Resources/translations/messages.de.po

```diff
@@ -2,884 +2,868 @@ msgid ""
+msgstr "Kunde <strong><a href=\"{{ url }}\">{{ Name }}}}</a></strong> modifiziert"
 
 msgid "Customer <strong>{{ name }}</strong> deleted"
-msgstr "(de) Customer <strong>{{ name }}</strong> deleted"
+msgstr "Kunde <strong>{{ Name }}}</strong> gelöscht"
 
 msgid "Customer detail"
-msgstr "(de) Customer detail"
+msgstr "Kundendetails"
 
 msgid "Customer not found"
-msgstr "(de) Customer not found"
+msgstr "Kunde nicht gefunden"
 
 msgid "Customer with e-mail address <strong>{{ email }}</strong> doesn't exist. <a href=\"{{ registrationLink }}\"> Register</a>"
-msgstr "(de) Customer with e-mail address <strong>{{ email }}</strong> doesn't exist. <a href=\"{{ registrationLink }}\"> Register</a>"
+msgstr "Kunde mit E-Mail-Adresse <strong>{{ E-Mail }}}</strong> existiert nicht. <a href=\"{{ registrationLink }}\"> Registrieren</a>"
 
 msgid "Date of subscription"
-msgstr "(de) Date of subscription"
+msgstr "Datum des Abonnements"
 
 msgid "Delivery address"
-msgstr "(de) Delivery address"
+msgstr "Lieferadresse"
 
 msgid "Delivery data"
-msgstr "(de) Delivery data"
+msgstr "Lieferdaten"
 
 msgid "Description"
-msgstr "(de) Description"
+msgstr "Beschreibung"
 
 msgid "Detail"
-msgstr "(de) Detail"
+msgstr "Detail"
 
 msgid "Discount"
-msgstr "(de) Discount"
+msgstr "Rabatt"
 
 msgid "Displaying"
-msgstr "(de) Displaying"
+msgstr "Anzeigen"
 
 msgid "Do you really want to remove this customer?"
-msgstr "(de) Do you really want to remove this customer?"
+msgstr "Möchten Sie diesen Kunden wirklich entfernen?"
 
 msgid "Do you want to receive the newest information about our shop? Subscribe to our e-mail newsletter."
-msgstr "(de) Do you want to receive the newest information about our shop? Subscribe to our e-mail newsletter."
+msgstr "Möchten Sie die neuesten Informationen über unseren Shop erhalten? Abonnieren Sie unseren E-Mail-Newsletter."
 
 msgid "Download the personal information"
-msgstr "(de) Download the personal information"
+msgstr "Persönliche Informationen herunterladen"
 
 msgid "E-mail"
-msgstr "(de) E-mail"
-
-msgid "E-mail settings:"
-msgstr "(de) E-mail settings:"
+msgstr "E-Mail"
 
 msgid "E-mail with a link to the export of your personal data was sent to your e-mail address."
-msgstr "(de) E-mail with a link to the export of your personal data was sent to your e-mail address."
+msgstr "E-Mail mit einem Link zum Export Ihrer persönlichen Daten wurde an Ihre E-Mail-Adresse gesendet."
 
 msgid "E-mail with a link to the page with your personal data was sent to your e-mail address."
-msgstr "(de) E-mail with a link to the page with your personal data was sent to your e-mail address."
+msgstr "E-Mail mit einem Link zur Seite mit Ihren persönlichen Daten wurde an Ihre E-Mail-Adresse gesendet."
 
 msgid "EAN"
-msgstr "(de) EAN"
+msgstr "EAN"
 
 msgid "Editing customer"
-msgstr "(de) Editing customer"
+msgstr "Kunden bearbeiten"
 
 msgid "Editing customer - %name%"
-msgstr "(de) Editing customer - %name%"
+msgstr "Kunden bearbeiten - %name%%%"
```

Comment:
The translation placeholders must be delimited by a single percent sign from both sides.